src/Chapter1.hs

```diff
@@ -640,8 +648,9 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n | div n 10 == 0 = n
+             | otherwise = firstDigit $ div n 10
```

Comment:
It looks like this function hangs on negative inputs. You can see how `div` behaves on them:

```haskell
ghci> div (-1) 10
-1
```